packages/image_picker/image_picker_platform_interface/lib/src/method_channel/method_channel_image_picker.dart

```diff
@@ -233,6 +233,57 @@ class MethodChannelImagePicker extends ImagePickerPlatform {
     return paths.map((dynamic path) => XFile(path as String)).toList();
   }
 
+  @override
+  Future<List<XFile>?> getMedia({
```

Comment:
Initially, to stay consistent with the interface and behavior of `getMultiImage`. However, now I looked into it a bit that implementation makes no sense. 

Looking at the Android implementation for `getMultiImage`, which `getMedia` partly uses, I think the idea was for `null` to be returned rather than an empty list, when the user explicitly cancels the flow. However, at least on my device, it is not possible to pick 0 items. It's always either a non-empty list, or `null`. 
As for iOS, there is no such logic and it results in the same: either a non-empty list, or `null`.

It too would make more sense to me for it to return `[]` rather than `null`, so we could make this non-nullable, but that would mean the behavior would have to change for `getMultiImage` as well, which I'm guessing to be a breaking change.

Reply:
It would be breaking for subclasses, yes. I think I'd rather be inconsistent than propagate the problematic use of a nullable collection to a new API though; less to clean up later.